tests/src/test_cases/widgets/test_dropdown.c

```diff
@@ -475,7 +475,8 @@ void test_dropdown_properties(void)
     TEST_ASSERT_EQUAL(2, lv_obj_get_property(obj, LV_PROPERTY_DROPDOWN_OPTION_COUNT).num);
 
     prop.id = LV_PROPERTY_DROPDOWN_SELECTED;
```

Comment:
`LV_PROPERTY_DROPDOWN_SELECTED` needs to update.
```
    LV_PROPERTY_ID2(DROPDOWN, SELECTED, LV_PROPERTY_TYPE_INT,  LV_PROPERTY_TYPE_BOOL, 0),

```

Reply:
Cherry picked the commit where you added the `LV_PROPERTY_ID2` define. Let's see if it's enough else i'll merge master

Reply:
This commit is already in master. Have you forgot to pull latest before rebasing?

```bash
git fetch https://github.com/lvgl/lvgl.git
git rebase -i FETCH_HEAD
```

Reply:
I did not forget. I rebased earlier and then you asked me to do something about all the files changed so i reverted the rebase to have a clean tree. I then needed the commit where you added `LV_PROPERTY_ID2` so i cherry picked that single commit

Am I missing something?

Reply:
>  i reverted the rebase to have a clean tree

Rebase on the latest lvgl master should have a clean tree, and no need to cherry-pick the property ID commit.

The code looks good to me by the way. Thank you!

Reply:
It should looks like this after rebasing. Three commit from your side.
```
commit 2d06026fdf00b9d237719449eaebc5067b141d80 (HEAD)
Author: André <andre_miguel_costa@hotmail.com>
Date:   Thu Dec 26 18:33:35 2024 +0100

    feat(dropdown): add second proprety to selected property

commit cd42f58202d46dd8e165fa0aa53539fd24c822e0
Author: André <andre_miguel_costa@hotmail.com>
Date:   Thu Dec 26 12:56:55 2024 +0100

    test(dropdown): add missing argument

commit 60b93a296a0e4bea228baa9587b0daf2dde34da9
Author: André <andre_miguel_costa@hotmail.com>
Date:   Fri Nov 15 19:38:34 2024 +0100

    BREAKING CHANGE(dropdown): add lv_anim_enable_t parameter to lv_dropdown_set_selected

commit 6b8a288437dd35d7e7a2d8592c49181cca2f6869 (lvgl/master)
Author: bjsylvia <bjsylvia@163.com>
Date:   Fri Dec 27 12:06:48 2024 +0800

    chore: fix typos in lv_draw_label.h (#7511)

```

Reply:
> The code looks good to me by the way. Thank you!

Thank you

> It should looks like this after rebasing. Three commit from your side.

You're right. I didn't force-push before after rebasing, rookie mistake :smiling_face_with_tear: